scripts/csv_to_parquet.py

```diff
@@ -0,0 +1,22 @@
+import os.path
+import sys
+
+import pyarrow.csv as pv
+import pyarrow.parquet as pq
+
+args = sys.argv
+
```

Comment:
Can you refector this script to have a main function?
Just to make the code a bit cleaner.
Can

Reply:
No problem